DQM/Integration/python/clients/scouting_dqm_sourceclient-live_cfg.py

```diff
@@ -0,0 +1,56 @@
+import FWCore.ParameterSet.Config as cms
+
+import sys
+if 'runkey=hi_run' in sys.argv:
+  from Configuration.Eras.Era_Run3_pp_on_PbPb_approxSiStripClusters_cff import Run3_pp_on_PbPb_approxSiStripClusters
+  process = cms.Process("DQM", Run3_pp_on_PbPb_approxSiStripClusters)
+else:
+  from Configuration.Eras.Era_Run3_2025_cff import Run3_2025
+  process = cms.Process("DQM", Run3_2025)
+
+unitTest = False
+if 'unitTest=True' in sys.argv:
+	unitTest=True
+
+if unitTest:
+  process.load("DQM.Integration.config.unittestinputsource_cfi")
+  from DQM.Integration.config.unittestinputsource_cfi import options
+else:
+  process.load("DQM.Integration.config.inputsource_cfi")
+  from DQM.Integration.config.inputsource_cfi import options
+
+  if not options.inputFiles:
+      process.source.streamLabel = "DQMOnlineScouting"
+
+process.load("DQM.Integration.config.environment_cfi")
+
+process.dqmEnv.subSystemFolder = 'ScoutingDQM'
+process.dqmSaver.tag = 'ScoutingDQM'
+process.dqmSaver.runNumber = options.runNumber
+# process.dqmSaverPB.tag = 'ScoutingDQM'
+# process.dqmSaverPB.runNumber = options.runNumber
+
+process.load("Configuration.StandardSequences.GeometryRecoDB_cff")
+process.load("Configuration.StandardSequences.MagneticField_cff")
+
+#---- for P5 (online) DB access
+process.load("DQM.Integration.config.FrontierCondition_GT_cfi")
+# Condition for lxplus: change and possibly customise the GT
+#from Configuration.AlCa.GlobalTag import GlobalTag as gtCustomise
+#process.GlobalTag = gtCustomise(process.GlobalTag, 'auto:run3_data', '')
+
+
+### for pp collisions
+process.load("DQM.HLTEvF.ScoutingCollectionMonitor_cfi")
+process.scoutingCollectionMonitor.topfoldername = "ScoutingDQM/"
+process.scoutingCollectionMonitor.onlyScouting = True
+process.dqmcommon = cms.Sequence(process.dqmEnv
+                               * process.dqmSaver)#*process.dqmSaverPB)
+
+process.p = cms.Path(process.dqmcommon * process.scoutingCollectionMonitor)
```

Comment:
@silviodonato @elfontan shall we run more than this module in the DQM sequence?
E.g. also `HLTriggerOffline/Scouting/python/ScoutingMuonTriggerAnalyzer_cfi.py` seems to be possible to run directly on the `DQMOnlineScouting` stream. Any other module? 
Thanks in advance.

Reply:
@mmusich @jprendi you can find the commit to add the MUO e JME DQM  in
https://github.com/silviodonato/cmssw/commit/c492bb0019c31c54f7a5c1606061f977ba770948

I managed to run it with
```
cmsRun scouting_dqm_sourceclient-live_cfg.py unitTest=True dataset=/ScoutingPFRun3/Run2025C-v1/HLTSCOUT runNumber=392175 minLumi=500 maxLumi=500
```

@mmusich do you think it is ok to append the missing `L1GtTriggerMenuRcd`  in this way?

Reply:
Right, I will make a JIRA to include the `hltOnlineMetaDataDigis`. 
We will need also need to add  `hltOnlineMetaDataDigis` as a fallback of  `onlineMetaDataDigis`